src/plugins/dataflow-tool/components/dataflow-program.tsx

```diff
@@ -717,14 +739,12 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
     // this.closeEditorNodePlots();
     // clearInterval(this.intervalHandle);
   };
-  private setProgramRunTime = (time: number) => {
-    this.props.onProgramRunTimeChange(time);
-  };
   private generateProgramData = (programTitle: string) => {
     let interval =  1;
     let datasetName = "";
     const programStartTime = Date.now();
-    const programEndTime = programStartTime + (1000 * this.props.programRunTime);
+    // const programEndTime = programStartTime + (1000 * this.props.programRunTime);
+    const programEndTime = programStartTime + 1000;
```

Comment:
A comment might be useful here -- does the notion of `programEndTime` even mean anything at this point?

Reply:
And I'm just suggesting that a comment in the code to that effect would be a simple way of acknowledging the issue without getting bogged down by it at this point.